lib/tasks/tenantAdd.js

```diff
@@ -145,6 +145,15 @@ function questions(done) {
                );
             },
          },
+         {
+            name: "url",
+            type: "input",
+            message: "Enter Tenant URL :",
+            default: "",
+            when: (values) => {
+               return !values.url && typeof Options.url == "undefined";
```

Comment:
`typeof Options.url == "undefined"`

Should validate when Options.url is `null` ?